Week1/MAKEME.md

```diff
@@ -7,108 +7,219 @@ Topics discussed this week:
 • API calls
 ```
 
-
 >[Here](/Week3/README.md) you find the readings you have to complete before the ninth lecture.
 
 ## Step 1: Feedback
 
-_Deadline Monday_
-
-Please provide feedback in an issue.
-
-_Deadline Monday_
+**_Deadline Monday_**
 
-## Step 2: FINISH ALL YOUR JAVASCRIPT HOMEWORK
+Please provide feedback on last week's homework from a fellow student as a GitHub issue.
 
-_Deadline Saturday_
+## Step 2: Single Page Application :sweat_drops:
 
-:point_up:
+**_Deadline Thursday_**
 
-## Step 3: SPA :sweat_drops:
+_This homework is more extensive and challenging than previous homework! Please read the instructions below carefully and follow them meticulously. Start this homework as soon as you can and allow time for discussion and questions (slack!)._
```

Comment:
maybe replace "meticulously" with "great attention to detail"

Reply:
Fixed